extensions/docker-desktop/README.md

```diff
@@ -1 +1,65 @@
 # Tanzu Community Edition extension for Docker Desktop
+
+This provides an extension integration with Docker Desktop to allow creating
+a local Tanzu Community Edition unmanaged cluster quickly and easily through
+the Docker Desktop interface.
+
+## Prerequisites
+
+In order to run this extension, you must have Docker Desktop 4.8.0 or later
+installed.
+
+If you would like to contribute or modify the extension, there are additional
+requirements. This extension is comprised of Go and React JavaScript code.
+Building the extension can be done using containerized build tools, but you may
+want to install development environments for these tools.
+
+Runtime Requirements:
+
+- [Docker Desktop 4.8.0 or later](https://www.docker.com/products/docker-desktop/)
+
+Development Recommendations:
+
+- [Go programming language](https://go.dev/doc/install)
+- [React reference](https://reactjs.org)
+- [Docker Extensions CLI](https://github.com/docker/extensions-sdk)
+
+## Building and Installing
+
+The standard way to get the Tanzu Community Edition extension for Docker
+Desktop is by using the Docker Marketplace. This will install the official
+released version of the extension.
+
+If you are making local changes and would like to try them out, you will need
+to follow these steps:
+
+1. In Docker Desktop, go to Preferences > Extensions and make sure
+   "Enable Docker Extensions" is checked.
+1. From a terminal, navigate to `$TCE_REPO/extensions/docker-desktop`.
+1. Run the following commands to build and install the local extension:
+
+   ```sh
+   make build-deps
```

Comment:
These 2 steps can be simplified to only `make extension`

Reply:
I got a bunch of failures when I tried to do that. Is there something else missing that needs to be done prior to this?

Reply:
Maybe I need to revisit the makefile. Can you publish containers to ghcr.io for this project? Is this something desirable?

Reply:
I do not know. Maybe that's something @joshrosso can weigh in on?

Reply:
Weird - looks like the `extension` target does both these steps:
https://github.com/vmware-tanzu/community-edition/blob/8434221ace624fad6f01b15d8c79de4ae5ee7f37/extensions/docker-desktop/Makefile#L45-L46

Reply:
Oooh... I was mixing up `make build-extension` and `make extension`. I will update.

We may want to consider renaming and cleaning up the targets to be be a little more clear. ;)